packages/react-scripts/template/README.md

```diff
@@ -430,8 +429,8 @@ Then we can change `start` and `build` scripts to include the CSS preprocessor c
 
 ```diff
    "scripts": {
-     "build-css": "node-sass src/ -o src/",
-     "watch-css": "npm run build-css && node-sass src/ -o src/ --watch --recursive",
+     "build-css": "node-sass-chokidar src/ -o src/",
+     "watch-css": "npm run build-css && node-sass-chokidar src/ -o src/ --watch --recursive",
```

Comment:
Do you mean the initial terminal output? If so yes you can pass -q to build-css like so: 
`npm run build-css -- -q`, this will call it in quiet mode. if this is not why you mean, create react app does not let you ATM add in custom loaders for scss, so this is a workaround to allow for that. In practice it has t been a big deal tbh